.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Module",
+            "type": "python",
+            "request": "launch",
+            "module": "ossdev"
+        }
+    ]
+}
```

Comment:
I am not familiar with tool creating this file. Is it common to version it? I mean `.idea` project files should not be in the repository so just checking if this is the case.

Reply:
I guess it depends on the preferred workflow of the main developers, but afaik yes, it is a shared configuration instructing VS Code how to launch/debug the project, making it work basically out-of-the-box in it.